### PR TITLE
updated Python wrapper

### PR DESCRIPTION
**Pull request by matkuki**
- update the type naming convention and also added the 'deprecate' pragmas for all old type names
- added a couple of missing procedures needed by the examples
- new examples, deleted the old ones
- renamed .babel to .nimble file

Everything was tested and working on windows. Feedback from other platforms would be much appreciated.

I have the wrapper for Python 3 ready and waiting, I was thinking it would be better to add it as a separate library. Do you agree?

Matic

### CODE REVIEW

**Comment by Araq** [comment]
> I have the wrapper for Python 3 ready and waiting, I was thinking it would be better to add it as a separate library. Do you agree?

I guess that makes sense.

**Comment by Araq** [comment]
If you change that much, please make it adhere to NEP-1 (our style guide) rather than what the Python C programmers considered good style (which is ugly).

**Comment by matkuki** [comment]
Are you sure, Araq?
Porting from C (manuals, tutorials, examples,...) will then become less trivial if function/constant/variable names change.

**Comment by Araq** [comment]
That's what we usually try to do for new wrappers.

**Comment by matkuki** [comment]
@Araq 
Updated to NEP-1 guidelines. Please review.

File: python.nimble [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Package]
+name: "python"
+version: "1.1"
+author: "Andreas Rumpf"
```
Comment (Araq) [inline python.nimble]: Replace mine name with yours or add yours. Give yourself some credit!

**Comment by matkuki** [comment]
@Araq 
Thanks. Also updated the version number.

File: examples/examples.nim [hunk 2]
```diff
@@ -0,0 +1,140 @@
+
+## Various Python 2 examples
+
+import python2
+import os, strutils
+
+
+## Very high level python binding examples
+proc Example1() =
```
Comment (dom96) [inline examples/examples.nim]: Procedures shouldn't start with uppercase letters.

File: python.nimble [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[Package]
+name: "python"
+version: "1.2"
+author: "Andreas Rumpf & Matic Kukovec"
```
Comment (dom96) [inline python.nimble]: Separate by comma not ampersand please.

**Comment by dom96** [comment]
`stmt` (and `expr`) is a special type, defining an enum value by that name may cause issues: https://github.com/matkuki/python/blob/master/src/python.nim#L200

**Comment by dom96** [comment]
I would also consider removing the `py` prefixes.

**Comment by matkuki** [comment]
@dom96 
Done, done, done and done! Compiles and works.
Please check.

**Comment by dom96** [comment]
Looks good! Thanks!

**Pull request merged**